plugins/generator-datapack-1.19.x/datapack-1.19.x/utils/defaultstates.json

```diff
@@ -0,0 +1,6328 @@
+{
```

Comment:
This file was not updated for 1.19.2.

If you just copy+paste the files, please at least let us know what you did not do so we know...

Reply:
I am now sceptical what in this PR was tested and what not, so please list features that you tested for sure, so I know what I still need to check. Thanks!

Reply:
Default states is used by custom biomes and dimensions which are not part of this PR. I didn't test it nor have the intention to port it.

Reply:
Ok